CMakePresets.json

```diff
@@ -20,7 +20,7 @@
       "description": "Configure with vcpkg toolchain",
       "binaryDir": "${sourceDir}/builds/${presetName}",
       "generator": "Ninja Multi-Config",
-      "toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
+      "toolchainFile": "$env{CMAKE_TOOLCHAIN_FILE}",
```

Comment:
Why did this change? I'm not sure what the **best** way of setting the toolchain is, but since this is a vcpkg preset, I feel like this should be calculated off of `VCPKG_ROOT` environment variable.

Using the environment `CMAKE_TOOLCHAIN_FILE` to set `toolchainFile` seems superfluous, given CMake already does this by default with the env variable:

> `CMAKE_TOOLCHAIN_FILE` is initialized by the [CMAKE_TOOLCHAIN_FILE](https://cmake.org/cmake/help/latest/envvar/CMAKE_TOOLCHAIN_FILE.html#envvar:CMAKE_TOOLCHAIN_FILE) environment variable if it is set when a new build tree is first created.

https://cmake.org/cmake/help/latest/variable/CMAKE_TOOLCHAIN_FILE.html

Reply:
hmm yeah i think you're right
was referencing what we did in remill https://github.com/lifting-bits/remill/blob/master/CMakePresets.json#L54 which might be wrong

Reply:
https://github.com/lifting-bits/gap/pull/43/commits/90adce05d8877a4b2fdb5aa5b4aac8abff4af6c0